lab 2/java/com/example/myapp/FourthActivity.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.myapp
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class FourthActivity : AppCompatActivity() {
+//        companion object {
+//        val GOT_DATA = Recipe("","",0, arrayListOf(""), arrayListOf(""))
+//    }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_fourth)

```

Comment:
Change init of all activities views to use android view binding https://developer.android.com/topic/libraries/view-binding